app/src/main/java/com/example/capstone/LoginFragment.java

```diff
@@ -87,6 +69,28 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         btnLogin = getView().findViewById(R.id.btn_login);
         btnSignup = getActivity().findViewById(R.id.btn_signup);
 
+        // Initialize Facebook Login button
```

Comment:
We want to avoid comments in code as much as possible.
This provides a good summary of why: 
https://medium.com/@bpnorlander/stop-writing-code-comments-28fef5272752
This lists some guidelines around comments:
https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules

In this case, creating a `initializeFacebookLogin()` method would be a better idea. So the method name explains your intent rather than the comment. This would also break down this big method into smaller ones.